src/middlewared/middlewared/plugins/system_general/update.py

```diff
@@ -45,33 +47,7 @@ class Config:
         datastore_extend = 'system.general.general_system_extend'
         cli_namespace = 'system.general'
         role_prefix = 'SYSTEM_GENERAL'
-
-    ENTRY = Dict(
-        'system_general_entry',
-        Patch(
-            'certificate_entry', 'ui_certificate',
-            ('attr', {'null': True, 'required': True, 'private': True}),
```

Comment:
Does having `private: True` for the `ui_certificate` field mean should be treated as secret type? If I'm correct, the new API schema doesn't mark this as Secret type.